frontends/election-manager/README.md

```diff
@@ -1,6 +1,10 @@
 # Election Manager (VxAdmin)
 
-This app is intended to be used on an offline computer by an election admin.
+This app is intended to be used on an offline computer by system administrators
+and election managers.
+
+Note: Election manager can refer to VxAdmin or a user role, depending on the
+context.
```

Comment:
lol 🤦

Reply:
do we have a plan to disambiguate?

Reply:
Yes! We're planning to rename `frontends/election-manager` as documented in https://github.com/votingworks/vxsuite/issues/2287 (also mentioned in https://github.com/votingworks/vxsuite/pull/2284#discussion_r940387472)